### PR TITLE
Simplify Ruby's before_send example

### PR DESCRIPTION
**Pull request by st0012**
Users can filter the entire event payload with Rails' parameter filter, which would make things a lot simpler.

### CODE REVIEW

File: src/includes/configuration/before-send/ruby.mdx [hunk 1]
```diff
@@ -2,13 +2,8 @@
...
   filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters)
   config.before_send = lambda do |event, hint|
-    request = event.request
-
-    if request
-      request.data = filter.filter(request.data)
-    end
-
-    event
+    # use Rails' parameter filter to senitize the event
```
Comment (imatwawana) [inline src/includes/configuration/before-send/ruby.mdx:5]: Fixing typo

```suggestion
    # use Rails' parameter filter to sanitize the event
```

**Review by imatwawana — APPROVED** [review]
Quick typo fix

File: src/includes/configuration/before-send/ruby.mdx (hunk 1 above)
Comment (st0012) [reply src/includes/configuration/before-send/ruby.mdx]: @imatwawana updated, thanks 👍

**Comment by PeloWriter** [comment]
@st0012 - do you need me to merge this, or can you?

**Comment by st0012** [comment]
@PeloWriter please merge this for me, thanks 🙂

**Comment by PeloWriter** [comment]
Done. Thanks @st0012

**Pull request merged**